### PR TITLE
Bug 1910111: order startup of IPsec components

### PR DESCRIPTION
**Pull request by markdgray**

### CODE REVIEW

File: bindata/network/ovn-kubernetes/ipsec.yaml [hunk 1]
```diff
@@ -182,7 +194,7 @@ spec:
           # Start the pluto IKE daemon and use the ipsec configuration file that
           # persists across container restarts. If the container has been restarted,
           # this will contain the previous configuration from ovs-monitor-ipsec
-          /usr/libexec/ipsec/pluto --leak-detective --config /etc/openvswitch/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
+          /usr/libexec/ipsec/pluto --leak-detective --config /etc/ipsec.conf --logfile /var/log/openvswitch/libreswan.log
```
Comment (squeed) [inline bindata/network/ovn-kubernetes/ipsec.yaml:197]: This doesn't match the above comment; this file isn't bind-mounted and thus won't be persisted, right?

Comment (markdgray) [reply bindata/network/ovn-kubernetes/ipsec.yaml:197]: Ooops. Forgot to update comment.

**Comment by markdgray** [comment]
Updated the comment as per @squeed 's request.

**Pull request merged**